### PR TITLE
WIP: add src group fetcher

### PR DESCRIPTION
**Pull request by ripx80**
initial code to fetch remote oidc groups

### CODE REVIEW

**Comment by szuecs** [comment]
please --sign-off commits, thanks (DCO)

File: filters/auth/oidc.go [hunk 1]
```diff
@@ -734,6 +735,57 @@ func (f *tokenOidcFilter) Request(ctx filters.FilterContext) {
...
+		payload := strings.NewReader("{\"securityEnabledOnly\": false}")
+		req, err := http.NewRequest("POST", urlParsed.String(), payload)
+		if err != nil {
+			log.Errorf("error constructing groups endpoint request: %s", err)
+			return
+		}
+		req.Header.Add("content-type", "application/json")
+		container.OAuth2Token.SetAuthHeader(req)
+
+		client := http.DefaultClient
```
Comment (szuecs) [inline filters/auth/oidc.go:763]: either use authclient via authclient.go or use our net/Client

Comment (universam1) [reply filters/auth/oidc.go:763]: /close

**Comment by szuecs** [comment]
close as requested

**Pull request closed without merging**